# ====== DO NOT MODIFY! AUTOGENERATED FROM AN ACTION DEFINITION ======
#feedback
geometry_msgs/PoseWithCovariance pose
geometry_msgs/TwistWithCovariance twist

